Addresses #2008 edit 'medium spiny neuron'

bvarner-ebi opened the pull request:
Addresses #2008 edit 'medium spiny neuron'

bvarner-ebi commented:
#gogoeditdiff

gouttegd reviewed (approved):
Looks good. Maybe consider adding a cross-reference to [this paper](https://doi.org/10.1073/pnas.242428599) on the definition?

bvarner-ebi commented:
> Looks good. Maybe consider adding a cross-reference to [this paper](https://doi.org/10.1073/pnas.242428599) on the definition?

Thank you for the reference. Added as a dbxref to the definition and added two synonyms.
